src/style.css

```diff
@@ -0,0 +1,64 @@
+@font-face {
+  font-family: Avenir;
+  src: url(./fonts/Avenir-Book.ttf);
+  font-weight: 300;
+}
+
+@font-face {
+  font-family: Avenir;
+  src: url(./fonts/Avenir-Heavy.ttf);
+  font-weight: 800;
+}
+
+body {
+  box-sizing: border-box;
+  font-family: Avenir, sans-serif;
+  font-weight: 300;
+  margin-top: 20px;
+  margin-bottom: 20px;
+}
+
+.bar {
+  font-family: inherit;
+  border-radius: 4px;
+  width: 100%;
+  background-image: url(images/Search.svg);
+  background-repeat: no-repeat;
+  border: 1px solid #e1e7ed;
+}
+
+.first {
```

Comment:
Avoid using numbers in selector names. They are not informative

Reply:
got it!

Reply:
My comment still not fixed.